maestro/trainer/models/florence_2/checkpoints.py

```diff
@@ -47,14 +49,17 @@ def load_model(
     processor = AutoProcessor.from_pretrained(model_id_or_path, trust_remote_code=True, revision=revision)
 
     if optimization_strategy == OptimizationStrategy.LORA:
-        config = LoraConfig(
-            r=8,
-            lora_alpha=16,
-            lora_dropout=0.05,
-            bias="none",
-            target_modules=["q_proj", "o_proj", "k_proj", "v_proj", "linear", "Conv2d", "lm_head", "fc2"],
-            task_type="CAUSAL_LM",
-        )
+        default_params = {
+            "r": 8,
+            "lora_alpha": 16,
+            "lora_dropout": 0.05,
+            "bias": "none",
+            "target_modules": ["q_proj", "o_proj", "k_proj", "v_proj", "linear", "Conv2d", "lm_head", "fc2"],
+            "task_type": "CAUSAL_LM",
+        }
+        if peft_advanced_params is not None:
+            default_params.update(peft_advanced_params)
```

Comment:
what will happen if `peft_advanced_params` will contain unsupported keys? maybe we should validate it?